convert/src/label_studio_to_badgerdoc/badgerdoc_format/annotation_converter.py

```diff
@@ -1,10 +1,28 @@
-from typing import List, Optional, Tuple
+from typing import Any, List, Optional, Tuple
 
-from .annotation_converter_practic import AnnotationConverterPractic
-from ..models.bd_annotation_model import AnnotationLink, BadgerdocAnnotation, Obj, Page, Size
 from ..models import bd_annotation_model_practic
+from ..models.bd_annotation_model import (
+    AnnotationLink,
+    BadgerdocAnnotation,
+    Obj,
+    Page,
+    Size,
+)
 from ..models.bd_tokens_model import Page as BadgerdocTokensPage
-from ..models.label_studio_models import LabelStudioModel, ModelItem, ResultItem
+from ..models.label_studio_models import (
+    LabelStudioModel,
+    ModelItem,
+    ResultItem,
+)
+from .annotation_converter_practic import AnnotationConverterPractic
+
+
+def safe_list_get (l: List, idx: int, default: Any) -> Any:
+    if l:
```

Comment:
empty list will return None despite the different (possible) value for arg `default`.

Reply:
good point - changed a logic a bit